src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -401,6 +409,37 @@ public void testSnapshotBuilder() throws Exception {
     List<String> searchConceptNames =
         searchConceptResponse.getResult().stream().map(SnapshotBuilderConcept::getName).toList();
     assertThat("expected concepts are returned", searchConceptNames, contains("concept1"));
+
+    var rollupCountsResponse =
+        dataRepoFixtures.getRollupCounts(
+            steward,
+            datasetId,
+            new SnapshotBuilderCountRequest()
+                .cohorts(
+                    List.of(
+                        new SnapshotBuilderCohort()
+                            .criteriaGroups(
+                                List.of(
+                                    new SnapshotBuilderCriteriaGroup()
+                                        .meetAll(true)
+                                        .mustMeet(true)
+                                        .criteria(
+                                            List.of(
+                                                new SnapshotBuilderProgramDataListCriteria()
+                                                    .values(List.of(0))
+                                                    .kind(SnapshotBuilderCriteria.KindEnum.LIST)
+                                                    .id(1),
+                                                new SnapshotBuilderProgramDataRangeCriteria()
+                                                    .high(1960)
+                                                    .low(1940)
+                                                    .kind(SnapshotBuilderCriteria.KindEnum.RANGE)
+                                                    .id(0),
+                                                new SnapshotBuilderDomainCriteria()
+                                                    .conceptId(1)
+                                                    .kind(SnapshotBuilderCriteria.KindEnum.DOMAIN)
+                                                    .id(19))))))));
+    // Count is 19 because of the fuzzy low counts.
+    assertThat(rollupCountsResponse.getResult().getTotal(), is(19));
```

Comment:
additionally, just a style thing, could be nice to make this count test its own method like Phil did with the other ones in his PR: https://github.com/DataBiosphere/jade-data-repo/pull/1662/files#diff-d73b047089637f4497d5947376822a2fd247755d50c84e4b2566e8ac1ae8abffR386